package.json

```diff
@@ -30,7 +30,7 @@
     "@babel/core": "^7.14.3",
     "@types/styled-components": "^5.1.9",
     "autoprefixer": "^10",
-    "classnames": "^2.3.1",
```

Comment:
revert this back to 2.3.1, that's a major update and it is not clear if it introduces any breaking changes.